### ISSUE TITLE
Aroma Gapps Installer

### ISSUE THREAD

**Issue by Kaustuv47** [SHRP/bootable_recovery#4]
- [ ] I am running an official build of TWRP, downloaded from https://twrp.me/Devices/
- [ ] I am running the latest version of TWRP
- [x] I have read the FAQ (https://twrp.me/FAQ/)
- [x] I have searched for my issue and it does not already exist

**Device codename**:  Beryllium
**TWRP version**: Unknown
**Shrp version**: 3.0 Stable

#### WHAT STEPS WILL REPRODUCE THE PROBLEM?
Problem arises when flashing AROMA Gapps package

#### WHAT IS THE EXPECTED RESULT?
Flasing the Aroma Zip will open a installer to install gapps of our choice

#### WHAT HAPPENS INSTEAD?
Installer stuck on a page saying "Starting AROMA installer version 2.70RC2"
![photo_2021-01-31 13 54 08 (1)](https://user-images.githubusercontent.com/36958031/106378996-dbfaed80-63ce-11eb-9b02-a83ed3f076f9.jpeg)

#### ADDITIONAL INFORMATION
ScreenSort provided of flashing AROMA Zip

`/tmp/recovery.log`: 
`dmesg`:

**Comment by epicX67** [comment]
@Kaustuv47 How much dumb he is

**Comment by epicX67** [comment]
@Kaustuv47 I already told this is not depend on the recovery itself. Poco build doesn't support aroma

**Comment by Kaustuv47** [comment]
> @Kaustuv47 I already told this is not depend on the recovery itself. Poco build doesn't support aroma

I have posted it before watching telegram. And there may exist work around as Gapps packages are open source. And, even one can create that type of package on their own if they have sufficient knowledge of underlying hardware and architecture.
Thank you.

**Comment by Kaustuv47** [comment]
I am very glad to use such a nice and well maintained recovery software. I thanks the whole shrp recovery team for their valuable effort.At the same time, It's not a very great statement from such a good developers team, it doesn't look great. One should admit that they don't know what makes the problem arise, instead telling it's not supported. If the problem doesn't exit on shrp side then say it's not a problem with our software. I shall ask the open gapps developers to make some changes which will make it run. Or I will try myself to find a fix. Thank you. On Jan 31, 2021 2:32 PM, epicX67 <notifications@github.com> wrote:
@Kaustuv47 How much dumb he is

**Comment by epicX67** [comment]
@Kaustuv47 1. First of all that comment accidentally sent by bot. I am talking about something in other matter with the Devs and you suddenly creates issue so when I replied Its directly sent to GitHub via bot. That's why deleted that comment

2. Opengapps can't help you with aroma. Because aroma isn't supported on newer devices.

**Comment by Kaustuv47** [comment]
Okay I got that.Thank you for your kind information.On Jan 31, 2021 3:36 PM, epicX67 <notifications@github.com> wrote:
@Kaustuv47 1. First of all that comment accidentally sent by bot. I am talking about something in other matter with the Devs and you suddenly creates issue so when I replied Its directly sent to GitHub via bot. That's why deleted that comment
Opengapps can't help you with aroma. Because aroma isn't supported on newer devices.
